icubam/messaging/telegram/integrator.py

```diff
@@ -0,0 +1,63 @@
+import tornado.queues
+from icubam.messaging.telegram import bot, updater, webhook
+
+
+class TelegramSetup:
+  """Sets up the telegram in the server support depending on the config."""
```

Comment:
Maybe you could link the the relevant Telegram API docs in the docsting? In particular I'm not sure I understand the conditions under which the webhook is used.

Reply:
Sure. will do.
Basically you have two ways to get the messages from telegram to our server:
1) periodically ask telegram to get the last messages.
2) set a webhook so that telegram forwards directly every message (call update) to us when they receive it. The webhook must be a https url.

The integrator decides if we are getting the messages via webhook or periodic callback and sets the right stuff in the message server. in case telegram is not supported by our config, it just does nothing (say you have no bot set up).